README.md

```diff
@@ -1,2 +1,34 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+**NOTE**: The Migration procedure will come with a downtime; As we need to scale down the applications using the Volumes before migration.
+## Prerequisite
+This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1) Minimum ceph version should be Ceph-CSI supported
+2) Existing volumes should be dynamically provisioned.
+3) Stop the application pods that are consuming the flex volume(s) that need to be converted.
```

Comment:
Instead of a prerequisite, let's make stopping the app the first step of the migration steps, then the last step is to start the application again.

Reply:
done